app/components/App.jsx

```diff
@@ -5,9 +5,9 @@ import Sidebar from './Sidebar'
 import Notifier from './Notifier'
 
 const App = ({ categories, children }) => (
-  <div role='application'>
+  <div className='con-wrapper coz-sticky'>
```

Comment:
ooooooh right. I just saw you remove it but not re-add it so i freaked out a little. Now that I can think of it, if there wasn't any role=application, you would know by now cause the layout wouldn't like it. My bad. 😄